internal/dinosql/gen_test.go

```diff
@@ -94,33 +86,8 @@ func TestColumnsToStruct(t *testing.T) {
 	}
 }
 
-var mockSettings GenerateSettings
-
-func init() {
-	mockSettings = GenerateSettings{
```

Comment:
We should also remove the mockSettings global variable from the MySQL tests. Individual tests should instead create the necessary settings themselves.